unime/src/routes/(app)/scan/+page@.svelte

```diff
@@ -102,12 +102,41 @@
     });
   };
 
+  const mockOfferRequest = () => {
+    state.set({
+      ...$state,
+      current_user_prompt: {
+        type: 'credential-offer',
+        issuer_name: 'State University',
+        logo_uri: undefined,
+        credential_configurations: {
+          0: {
+            display: [
+              {
+                name: 'Graduation Diploma',
+                locale: 'en-US',
+              },
+            ],
+            credential_definition: {
+              type: ['VerifiableCredential', 'University Degree'],
+            },
+          },
+          1: {
+            credential_definition: {
+              type: ['VerifiableCredential', 'Cafeteria Voucher'],
+            },
+          },
+        },
+      },
+    });
+  };
```

Comment:
Accepting this Credential Offer results in a panic in the backend because of an `unwrap`.

This can be fixed by replacing this:
```rust
        let credential_offer = state.core_utils.active_credential_offer.unwrap();
```

for something like this:
```rust
        let credential_offer = state
            .core_utils
            .active_credential_offer
            .ok_or(AppError::Error("Missing active credential offer".to_string()))?;
```

In normal mode this `unwrap` is impossible to reach, but catching the error anyway is still the best solution.

Accepting the mock Credential Offer will then result in an error message.

Reply:
My solution seems to work the first time, but it seems that there is a 'semi-freeze' when selecting the mock Credential Offer for a second time. The first time it redirects to the "me" page, but not the second time. It's impossible to close the Error pop-up and therefore also impossible to click the "Reject" button:

![image](https://github.com/user-attachments/assets/55aae209-193f-48af-add7-89247186b92a)

So the current panic is not a good option, but my suggestion does not completely solve it. 

I can help to do some more debugging.

Reply:
All mock scans are removed now since they're obsolete and misleading. They should eventually be replaced with actual test calls with dynamic data.